saleor/plugins/webhook/tests/test_webhook.py

```diff
@@ -618,6 +619,39 @@ def test_checkout_updated(
     )
 
 
+@freeze_time("1914-06-28 10:50")
+@mock.patch("saleor.checkout.calculations.fetch_checkout_prices_if_expired")
+@mock.patch("saleor.plugins.webhook.plugin._get_webhooks_for_event")
+@mock.patch("saleor.plugins.webhook.plugin.trigger_webhook_sync")
+def test_get_shipping_methods_for_checkout_untaxed(
+    mocked_webhook_trigger,
+    mocked_get_webhooks_for_event,
+    mocked_fetch,
+    any_webhook,
+    settings,
+    checkout,
+    app,
+    permission_manage_shipping,
+):
+    app.permissions.add(permission_manage_shipping)
+    app.save()
```

Comment:
Don't think that you need to call `.save()` when you're adding external  relations.